scripts/deploy.py

```diff
@@ -268,19 +313,49 @@ def _execute_deployment():
 
         print 'Returning to oppia/ root directory.'
 
+    release_version_library_url = (
+        'https://%s-dot-%s.appspot.com/library' %
+        current_release_version, APP_NAME)
+    answer = check_errors_in_a_page(
```

Comment:
This is not an `answer` any more -- let's name it differently. Maybe call it library_page_loads_correctly.

Ditto below for other "answers".

Reply:
Done.